ndnsd/communication/sync-adapter.cpp

```diff
@@ -1,9 +1,9 @@
 /* -*- Mode:C++; c-file-style:"gnu"; indent-tabs-mode:nil; -*- */
 /*
- * Copyright (c) 2014-2019,  The University of Memphis
+ * Copyright (c) 2014-2020,  The University of Memphis
  *
  * This file is part of NDNSD.
- * See AUTHORS.md for complete list of NDNSD authors and contributors.
+ * See NLSR's AUTHORS.md for complete list of NDNSD authors and contributors.
```

Comment:
for this file, perhaps just keep the NLSR's copyright, with copyright year update.

Reply:
done.